packages/data-controls/src/test/index.js

```diff
@@ -62,13 +68,13 @@ describe( 'controls', () => {
 			selectorWithFalseResolver.mockReturnValue( 'bar' );
 			selectorWithResolver.mockReturnValue( 'resolved' );
 			hasFinishedResolution.mockReturnValue( false );
+			resolveSelectSelectorWithResolver.mockResolvedValue( 'resolved' );
 		} );
 		afterEach( () => {
 			selectorWithUndefinedResolver.mockClear();
 			selectorWithResolver.mockClear();
 			selectorWithFalseResolver.mockClear();
 			hasFinishedResolution.mockClear();
-			unsubscribe.mockClear();
```

Comment:
Shouldn't we clear the new mocks here as well?

Reply:
We are using the same mocks now so it is not needed anymore.